ui/src/views/Home.vue

```diff
@@ -19,42 +19,52 @@
 import axios from "axios";
 
 import Alert from "@/components/Layout/Alert.vue";
+import { store } from "@/store.js";
 
 export default {
   components: {
     Alert
   },
   created() {
     this.config = require("@/config/config.json");
+    if(store.state.allowToAccessTheNextPage)
```

Comment:
Isn't this variable going away now? Isn't it all handled by the router at this point?